resources/user.go

```diff
@@ -196,12 +201,27 @@ func (obj *UserRes) CheckApply(apply bool) (checkOK bool, err error) {
 		if obj.GID != nil {
 			args = append(args, "-g", fmt.Sprintf("%d", *obj.GID))
 		}
+		if obj.Group != nil {
+			args = append(args, "-g", *obj.Group)
```

Comment:
we should probably check for invalid chars in this obj.Group string in Validate(). Even though it's directly being passed into argv, it may change, thus exposing us for a cli injection attack. Best to filter out bogus things like spaces, quotes, and anything else that's illegal.